Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -84,6 +84,7 @@ public static void PrintF(byte* str, void* varargs)
         }
 
         Console.Write(formatString.Substring(currentPosition));
+        return 0;
```

Comment:
Getting a proper return code is tricky. You can make this `return 1` and open a follow-up issue to make the implementation to return a correct number.